test/unit/dockerfile_constructors_test.lua

```diff
@@ -34,20 +34,13 @@ local function check_output(command, ...)
     return nil, cmd_err
 end
 
-local function get_install_tarantool_enterprise_layers(sdk_version)
+local function get_copy_tarantool_enterprise_layers()
     local layers = remove_leading_spaces([=[
-        ### Install Tarantool Enterprise
-        ARG DOWNLOAD_TOKEN
-
-        RUN DOWNLOAD_URL=https://tarantool:${DOWNLOAD_TOKEN}@download.tarantool.io \
-            && mkdir -p /usr/share/tarantool \
-            && cd /usr/share/tarantool \
-            && curl -O -L ${DOWNLOAD_URL}/enterprise/tarantool-enterprise-bundle-SDK_VERSION.tar.gz \
```

Comment:
Could we remove a "curl" from Dockerfile after that?

Reply:
What did you mean?

Reply:
Is there a place inside Dockerfile there you install "curl"?

Reply:
No

Reply:
Ok, my question is not actual